services/director-v2/src/simcore_service_director_v2/modules/dask_clients_pool.py

```diff
@@ -33,13 +33,13 @@ def __post_init__(self):
         self._client_acquisition_lock = asyncio.Lock()
 
     @staticmethod
-    def default_cluster(settings: DaskComputationalBackendSettings):
+    def default_cluster(settings: ComputationalBackendSettings):
         return Cluster(
-            id=settings.DIRECTOR_V2_DEFAULT_CLUSTER_ID,
+            id=settings.COMPUTATIONAL_BACKEND_DEFAULT_CLUSTER_ID,
             name="Default cluster",
             type=ClusterType.ON_PREMISE,
```

Comment:
i"m curious: How is this pg-model used? I see we have `on_premise` and `aws`. What are the implications?

Reply:
at the moment there are none. this is a descriptive only type at the moment.